robot_skills/src/robot_skills/api.py

```diff
@@ -7,8 +7,11 @@
 
 
 class Api(RobotPart):
-    def __init__(self, robot_name, tf_listener):
+    def __init__(self, robot_name, tf_listener, pre_hook=None, post_hook=None):
```

Comment:
Some docstring on this would be nice. I guess it's a function, taking what arguments?